docs/src/modules/components/MarkdownElement.js

```diff
@@ -403,6 +403,14 @@ const Root = styled('div')(
       opacity: 1,
       transform: 'translate(1px, 0)',
     },
+    '& a.remove-link-arrow[target="_blank"]::after': {
+      // Allows to remove link arrows for images
+      display: 'none',
+    },
+    '& a[href*="carbonads"][target="_blank"]::after': {
```

Comment:
It can be more than carbon ads, I think we need a more generic selectors.

Reply:
What about targeting links inside those 3 selectors:
```
document.querySelector('.ea-placement')
document.querySelector('#carbonads')
document.querySelector('.carbonads')
```

https://github.com/mui/material-ui/blob/master/docs/src/modules/components/Ad.js/#L157-L160

Reply:
Could we create add a new class name to the DOM instead? Making it less brittle to future changes. For example, what if we implement: https://www.notion.so/mui-org/docs-infra-Ad-networks-fcd294a093354a94a329ab9852433fdc?pvs=4#ee1fd00fd3b642f28439c7d1f211695f

Reply:
Do you know what is `.ea-placement` for? The search in codebase returns 0 files with this text 🤔

Could it be an old elements that does not exist anymmore?

Reply:
ed was for EthicalAds